From 7fe3924c859b247a030642cd436cc4611d1f25a0 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 14 Apr 2025 22:51:33 +0800
Subject: [PATCH] libarchive: pass correct cflags

---
 packages/libarchive.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libarchive.cmake b/packages/libarchive.cmake
index 0d7fa80..ae1c0fa 100644
--- a/packages/libarchive.cmake
+++ b/packages/libarchive.cmake
@@ -37,6 +37,7 @@ ExternalProject_Add(libarchive
         -DENABLE_TEST=OFF
         -DWINDOWS_VERSION=WIN10
         -DCMAKE_POLICY_VERSION_MINIMUM=3.5
+        "-DCMAKE_C_FLAGS='-lxml2 -lbz2 -llzo2 -lz -lbrotlienc -lbrotlidec -lbrotlicommon -lzstd -lws2_32 -lgdi32 -lcrypt32 -liconv -lbcrypt'"
     BUILD_COMMAND ${EXEC} ninja -C <BINARY_DIR>
     INSTALL_COMMAND ${EXEC} ninja -C <BINARY_DIR> install
     LOG_DOWNLOAD 1 LOG_UPDATE 1 LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1
-- 
2.48.1.windows.1

